tutorial/netstd/Server/Program.cs

```diff
@@ -272,6 +280,8 @@ private enum Transport
             Tcp,
             TcpBuffered,
             NamedPipe,
+            NamedPipeBuffered,
+            NamedPipeFramed,
             Http,
             TcpTls,
             Framed
```

Comment:
(Understand this isn't part of new code in the PR, however:) @Jens-G Why is "Framed" not "TcpFramed"?  Seems out of place.  Also can one have TcpTlsBuffered, TcpTlsFramed?  Thanks.

Reply:
I can't speak to reason for the existing setup but I know you can have TcpTlsBuffered and TcpTlsFramed as I've used both in testing or production.